Wireframe/css/style.css

```diff
@@ -0,0 +1,71 @@
+*{
+    text-align: center;
+    font-family: 'Courier New', Courier, monospace;
+    
+   
+}
+
+header{
+    background-color: rgba(255, 98, 0, 0.432);
+    color: white;
+    padding: 9px;
+    margin-bottom: 10px;
+}
+
+main{
+    display: grid;
+    gap:10px;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: auto auto;
+}
+
+.one{
```

Comment:
You mentioned that it was a bit cumbersome to manage the position of the elements inside the box.
Luckily, it's a perfect opportunity to use `grid` or `flex` layout here!

Depending on the specific situation it can be one or another.

Just a quick comparison of when to use CSS Grid versus Flexbox:
Use CSS Grid when:
1. Creating two-dimensional layouts (rows and columns)
2. Designing complex page layouts or overall page structures
3. You need precise control over both row and column alignment
4. Creating grid-based designs with gaps between elements

Use Flexbox when:
1. Working with one-dimensional layouts (either row or column)
2. Aligning items within a container
3. Distributing space among items in a single row or column
4. Creating flexible components or navigation menus

Grid is generally better for overall page layouts, while Flexbox excels at aligning content within components. Often, you'll use both in a project - Grid for the main layout and Flexbox for smaller component layouts.